app/javascript/components/pages/Home.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import {
+  Carousel,
+  CarouselItem,
+  CarouselControl,
+  CarouselIndicators,
+  CarouselCaption,
+} from 'reactstrap';
+
+const items = [
```

Comment:
Consider putting items in another file and then importing like you would with mock data to clean up the home page.